src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +61,16 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+
+            $shortNamePath = $resourceMetadata->getShortName() ?? '';
```

Comment:
@soyuka Having a default resource short name as an empty string is useless. Harmful even... :)